pkg/kubelet/volumemanager/populator/desired_state_of_world_populator_test.go

```diff
@@ -150,9 +159,103 @@ func TestFindAndAddNewPods_WithRescontructedVolume(t *testing.T) {
 
 }
 
+func TestFindAndAddNewPods_WithDifferentConditions(t *testing.T) {
+	tests := []struct {
+		desc          string
+		hasAddedPods  bool
+		podState      string
+		expectedFound bool // Found pod is added to DSW
+	}{
+		{
+			"HasAddedPods is false, ShouldPodRuntimeBeRemoved and ShouldPodContainerBeTerminating are both true",
+			false,
+			Removed,
+			false, // Pod should not be added to DSW
+		},
```

Comment:
it may just be me but I prefer field names to be present in struct instantiations..

Reply:
Updated. I do not have preference but I do what makes my reviewer feel they can read easier. :D